From: Feichtmeier <frederik.feichtmeier@gmail.com>
Date: Wed, 19 Feb 2020 10:36:23 +0100
Subject: Style looking-glass $variant independant

- use osd colors
- also fix the toolbar buttons jumping on click
- adding better unchecked/hover/checkedifferences
- use $link_color for links
- remove the strong text-shadow

Origin: https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/1023
Applied-Upstream: 3.35.92
---
 .../gnome-shell-sass/widgets/_looking-glass.scss   | 37 ++++++++++++----------
 1 file changed, 21 insertions(+), 16 deletions(-)

diff --git a/data/theme/gnome-shell-sass/widgets/_looking-glass.scss b/data/theme/gnome-shell-sass/widgets/_looking-glass.scss
index ad1ed90..9c38e63 100644
--- a/data/theme/gnome-shell-sass/widgets/_looking-glass.scss
+++ b/data/theme/gnome-shell-sass/widgets/_looking-glass.scss
@@ -5,14 +5,14 @@
   background-color: $osd_bg_color;
   spacing: $base_spacing;
   padding: 4px;
-  border: 2px solid transparentize($osd_fg_color, 0.8);
-  border-top-width:0;
-  border-radius: 0 0 $base_border_radius $base_border_radius;
+  border: 1px solid transparentize($osd_fg_color, 0.8);
+  border-radius: $base_border_radius;
+  color: $osd_fg_color;
 
   & > #Toolbar {
     border: none;
     border-radius: $base_border_radius;
-    background-color: darken($osd_bg_color, 10%);
+    background-color: $osd_bg_color;
   }
 
   .labels { spacing: $base_spacing; }
@@ -20,19 +20,18 @@
     -natural-hpadding: $base_padding * 2;
     -minimum-hpadding: 6px;
     font-weight: bold;
-    color: $fg_color;
+    color: darken($osd_fg_color, 15%);
     transition-duration: 100ms;
     padding-left: .3em;
     padding-right: .3em;
+    border-bottom-width: 2px;
     &:hover {
-      color: white;
-      text-shadow: black 0px 2px 2px;
+      color: $osd_fg_color;
     }
     &:selected {
       border-bottom-width: 2px;
-      border-color: lighten($selected_bg_color,5%);
-      color: white;
-      text-shadow: black 0px 2px 2px;
+      box-shadow: inset 0 -2px 0 0 lighten($selected_bg_color, 5%);
+      color: $osd_fg_color;
     }
   }
   StBoxLayout#EvalBox { padding: 4px; spacing: $base_spacing; }
@@ -41,12 +40,17 @@
 
 .lg-dialog {
   StEntry {
-    selection-background-color: #bbbbbb;
-    selected-color: $osd_bg_color;
+    background-color: transparentize(lighten($osd_bg_color, 5%), 0.4);
+    color: $osd_fg_color;
+    border-color: transparentize($osd_fg_color, 0.8);
+    min-height: 22px;
+    selection-background-color: $selected_bg_color;
+    selected-color: $selected_fg_color;
   }
   .shell-link {
-    color: #999999;
-    &:hover { color: #dddddd; }
+    color: $link_color;
+    &:hover { color: lighten($link_color, 10%); }
+    &:active { color: darken($link_color, 10%); }
    }
 }
 
@@ -60,7 +64,7 @@
 }
 
 .lg-obj-inspector-button {
-    border: 1px solid gray;
+    border: 1px solid $osd_borders_color;
     padding: 4px;
     border-radius: $base_border_radius;
     &:hover { border: 1px solid #ffffff; }
@@ -75,7 +79,8 @@
 }
 
 .lg-extension {
-    border: 1px solid $osd_borders_color;
+    border: 1px solid lighten($osd_borders_color, 5%);
+    background-color: lighten($osd_bg_color, 5%);
     border-radius: $base_border_radius;
     padding: 4px;
 }
